ajax/libs/froala-design-blocks/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "froala-design-blocks",
+  "version": "1.0.0-alpha.1",
```

Comment:
As CI said, we need to remove this version field.
After this PR be merged, CDNJS bot will add all files of every version.
Thanks.